plugin/output/postgres/postgres.go

```diff
@@ -238,6 +241,17 @@ func (p *Plugin) Start(config pipeline.AnyConfig, params *pipeline.OutputPluginP
 	}
 	p.pool = pool
 
+	stdBackoff := backoff.NewExponentialBackOff()
```

Comment:
maybe move this in function ? 
This will make it easier to change settings if needed.

Reply:
also need fix conflicts in pr

Reply:
done